src/providers/helpers/uploadObservation.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+import inatjs from "inaturalistjs";
+
+import { getJWTToken } from "../../components/LoginSignUp/AuthenticationService";
+import { markUploaded, markPhotoUploaded, markSoundUploaded } from "./markUploaded";
```

Comment:
This is a total style issue, but if I'm not trying to separate major functional groups like MVC components, I generally prefer to keep relevant code in the same file / module *unless* I need to reuse them. Here, these methods are only called from this module, so I think they can be included here.